packages/language-chooser/src/style.scss

```diff
@@ -0,0 +1,141 @@
+/**
+ * Colors
+ */
+
+// WordPress grays.
+$black: #000; // Use only when you truly need pure black. For UI, use $gray-900.
+$gray-900: #1e1e1e;
```

Comment:
Could we use the pre-existing `--wp-components-color-gray-` theme colors somehow?

`$gray-900` is also deprecated there in favor of `foreground` and we have `background` that uses white.